robottelo/constants.py

```diff
@@ -529,6 +529,12 @@
 CUSTOM_RPM_REPO = (
     u'http://repos.fedorapeople.org/repos/pulp/pulp/fixtures/rpm/'
 )
+CUSTOM_MODULE_STREAM_REPO_1 = (
+    u'http://mirror.math.princeton.edu/pub/fedora/linux/releases/28/Modular/x86_64/os/'
```

Comment:
This is an external url, are we sure it will be always available

Reply:
@omkarkhatavkar I am guessing
```https://dl.fedoraproject.org/pub/fedora/linux/updates/29/Modular/x86_64/```
or 
```https://repos.fedorapeople.org/pulp/pulp/fixtures/rpm-with-modules/``` is better

Reply:
@parthaa thanks for confirming, I will update the PR with the same changes.

Reply:
@ldjebran @parthaa and @pondrejk  updated all code based on new repos. please have a review